tutorials/generate-events.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: page
+title: Generating Events
+---
```

Comment:
I think this tutorial needs more context:

* What are the prerequisites for running the steps? What previous tutorials does this assume I have completed?
* What does this accomplish?
* What will I learn from this tutorial?